dev-meta.yaml

```
@@ -2,7 +2,7 @@ from:
 - source: registry-proxy.engineering.redhat.com/rh-osbs/openshift-golang-builder\:v(?:[\.0-9\-]*).*
   target: registry.ci.openshift.org/openshift/release:rhel-8-release-golang-1.19-openshift-4.12 AS builder
 - source: registry.redhat.io/ubi8:8.(\d)-([\.0-9])*
-  target: docker.io/centos:8 AS centos
+  target: registry.redhat.io/ubi8/ubi:8.7-929
```

Both `/ubi8/ubi` and `/ubi8` seem to refer to the same repository, but `/ubi8` seems to be the "canonical source" (see [here](https://catalog.redhat.com/software/containers/ubi8/5c647760bed8bd28d0e38f9f?tag=8.7-1090&push_date=1676991396000)). Do you know where the `/ubi8/ubi` path in this PR comes from?

```suggestion
  target: registry.redhat.io/ubi8:8.7-1090
```

Another idea: Because we are not actually replacing anything anymore, maybe it's also an idea to remove this replacement completely.